stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -113,6 +114,14 @@ public fun DefaultMessageItem(
     onLongItemClick: (Message) -> Unit = {},
     onThreadClick: (Message) -> Unit = {},
     onImagePreviewResult: (ImagePreviewResult?) -> Unit = {},
+    systemMessageContent: @Composable (SystemMessageItem) -> Unit = {
+        SystemMessage(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 12.dp, start = 16.dp, end = 16.dp, bottom = 12.dp),
```

Comment:
How about using the `vertical` and `horizontal` parameters instead like the below?

```kotlin
.padding(vertical = 12.dp, horizontal = 16.dp)
```

Reply:
Yep, sorry, was experimenting with the vales and forgot to replace the parameters back.